assets/prototype/application/utilities/money/test/useMoneyDisplay.test.ts

```diff
@@ -0,0 +1,26 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import useMoneyDisplay from '../useMoneyDisplay';
+import { mockEeJsData } from '../../../services/config/test/data';
+import { ApolloMockedProvider } from '../../../../domain/eventEditor/context/test';
+
+describe('useMoneyDisplay', () => {
+	const wrapper = ApolloMockedProvider();
+
+	it('checks for existence of properties', () => {
+		const { result } = renderHook(() => useMoneyDisplay(), { wrapper });
+
+		expect(result.current).toHaveProperty('beforeAmount');
+		expect(result.current).toHaveProperty('afterAmount');
```

Comment:
can we also test the values for these properties based on the mock JS data?

Reply:
Those are calculated, which mean those calculation will need to be pulled/duplicated into tests